### PR TITLE
added acpc.1.1.1

### PR DESCRIPTION
**Pull request by UnixJunkie**
fix compilation of 1.1.0 and remove version constraint on the dolog lib

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
Looks like a missing dependency:
```
#=== ERROR while installing acpc.1.1.1 ========================================#
# opam-version         1.2.2 (aa258ecc06d3aea5a67f442a4ffd23f2a457180b)
# os                   linux
# command              obuild configure
# path                 /home/opam/.opam/4.05.0/build/acpc.1.1.1
# compiler             4.05.0
# exit-code            9
# env-file             /home/opam/.opam/4.05.0/build/acpc.1.1.1/acpc-7-fbcac7.env
# stdout-file          /home/opam/.opam/4.05.0/build/acpc.1.1.1/acpc-7-fbcac7.out
# stderr-file          /home/opam/.opam/4.05.0/build/acpc.1.1.1/acpc-7-fbcac7.err
### stdout ###
# Configuring ACPC-1.0...
### stderr ###
# error: missing dependency 'bitv'
```

File: packages/acpc/acpc.1.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+homepage: "https://github.com/UnixJunkie/ACPC"
+dev-repo: "git://github.com/UnixJunkie/ACPC"
+bug-reports: "https://github.com/UnixJunkie/ACPC/issues"
+license: "BSD"
+build: [
+  ["obuild" "configure"]
+  ["obuild" "build"]
+]
+install: [
+  ["cp" "dist/build/acpc/acpc"
```
Comment (kit-ty-kate) [inline packages/acpc/acpc.1.1.1/opam]: This might be better replaced by a `acpc.install` file (that would be located in `acpc/acpc.1.1.1/files`)

**Comment by UnixJunkie** [comment]
Added missing bitv dep in opam file.
I will not add a .install file, because this will be generated automatically once this package will be put under jbuilder, which will have to wait for a later version.

**Comment by kit-ty-kate** [comment]
Seems iike this package requires OCaml >= 4.02
```
#=== ERROR while installing acpc.1.1.1 ========================================#
# opam-version 1.2.2
# os           linux
# command      obuild build
# path         /home/travis/.opam/4.01.0/build/acpc.1.1.1
# compiler     4.01.0
# exit-code    6
# env-file     /home/travis/.opam/4.01.0/build/acpc.1.1.1/acpc-21048-03b1a4.env
# stdout-file  /home/travis/.opam/4.01.0/build/acpc.1.1.1/acpc-21048-03b1a4.out
# stderr-file  /home/travis/.opam/4.01.0/build/acpc.1.1.1/acpc-21048-03b1a4.err
### stdout ###
# [ 1 of 20] Compiling My_utils                      
# [ 2 of 20] Compiling Amino_acid                    
### stderr ###
# Options are:
# [...]
#   -dalloc  (undocumented)
#   -dreload  (undocumented)
#   -dscheduling  (undocumented)
#   -dlinear  (undocumented)
#   -dstartup  (undocumented)
#   - <file>  Treat <file> as a file name (even if it starts with `-')
#   -help  Display this list of options
#   --help  Display this list of options
```
> I will not add a .install file, because this will be generated automatically once this package will be put under jbuilder, which will have to wait for a later version.

Why not doing it right now ? I mean it's just a simple file to put in the files directory. I can do it for you if you don't have time to do it.

**Comment by UnixJunkie** [comment]
It is another file to maintain by hand, separated from the opam file. So yes, I don't like to do it (it scatters the package's description). If you want to add it yourself, feel free to do so.

**Comment by UnixJunkie** [comment]
related to https://github.com/ocaml/opam-repository/pull/12093

**Comment by mseri** [comment]
The few remaining failure are infrastructure related. I suggest to merge this by hand and replace the install stanza with an install file in the merge, what do you think @kit-ty-kate ?

**Comment by kit-ty-kate** [comment]
oops yes it is. Sorry I missed your ping. Thanks a lot!

**Pull request merged**